maav-init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I pulled your repo. This file is not runnable. Please run `chmod +x` on it

Reply:
+1 to Ryan's comment

Reply:
This is done